### PR TITLE
groovy update: regenerate

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: graphene-sys/src/lib.rs [hunk 1]
```diff
@@ -333,6 +333,7 @@ impl ::std::fmt::Debug for graphene_vec4_t {
     }
 }
 
+#[link(name = "graphene-1.0")]
```
Comment (sdroege) [inline graphene-sys/src/lib.rs:336]: `Cargo.toml` should also have a `link = "graphene-1.0"` line

**Review by sdroege — COMMENTED** [review]
Looks good to me otherwise

**Review by sdroege — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Restarted CI.

**Comment by sdroege** [comment]
Needs some fixes for the CI.

**Review by sdroege — APPROVED** [review]

**Comment by bilelmoussaoui** [comment]
Looks good to me, I'm going to go ahead and merge it! 

Thanks for the review

**Pull request merged**